lib/chat/user_message_rows.dart

```diff
@@ -90,65 +77,100 @@ buildMessageRows(
+                    height: 60.0, 
+                    decoration: BoxDecoration(
+                      color: Colors.black, 
+                      shape: BoxShape.circle
+                    )
                   )
                 )
-              )
-            ),
-            Expanded(
-              child: Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: <Widget>[
-                  FormatText(
-                    text: recipient.fullName,
-                    fontSize: 14.0,
-                  ),
-                  RichText(
-                    overflow: TextOverflow.ellipsis,
-                    text: TextSpan(
-                      style: GoogleFonts.muli(
-                        textStyle: TextStyle(
-                          color: Colors.grey,
-                          letterSpacing: .5, 
-                          fontSize: 12.0, 
-                        )
-                      ),
-                      children: [
-                        if(isSent != false) WidgetSpan(
-                          child: Container(
-                            margin: EdgeInsets.only(right: 6.0),
-                            child: Icon(
-                              Icons.send, 
-                              size: 12, 
-                              color: Colors.grey
-                            )
-                          ),
+              ),
+              Expanded(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: <Widget>[
+                        FormatText(
+                          text: recipient.fullName,
+                          fontSize: 14.0,
                         ),
-                        TextSpan(
-                          text: lastMessageType == 0 ? 
-                            lastMessage.content : 
-                            imageMessageDesciption
+                        Container(
+                          padding: EdgeInsets.only(right: 10.0),
+                          child: FormatText(
+                            text: DateFormat('jm').format(
+                              DateTime.fromMillisecondsSinceEpoch(
+                                int.parse(lastMessage.timestamp)
+                              )
+                            ),
+                            fontSize: 14.0,
+                            fontWeight: FontWeight.w400,
+                            textColor: Colors.grey,
+                          )
                         )
-                      ]
+                      ],
                     ),
-                  )
-                ]
+                    RichText(
+                      overflow: TextOverflow.ellipsis,
+                      text: TextSpan(
+                        style: GoogleFonts.muli(
```

Comment:
`FormatText` widget can be used here too.

Reply:
this needs to be a `TextSpan` whereas the `FormatText` is a `Text` because this needs to be able to be inserted as part of the `RichText`